### PR TITLE
persist discovered brokers

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

File: flow/src/main/scala/org/alephium/flow/io/DatabaseVersion.scala [hunk 1]
```diff
@@ -21,10 +21,12 @@ import akka.util.ByteString
...
-final case class DatabaseVersion(value: Int) extends AnyVal
+final case class DatabaseVersion(value: Int) extends AnyVal with Ordered[DatabaseVersion] {
+  override def compare(that: DatabaseVersion): Int = value.compare(that.value)
+}
 
 object DatabaseVersion {
   implicit val serde: Serde[DatabaseVersion] = Serde.forProduct1(apply, _.value)
 
-  val currentDBVersion: DatabaseVersion = DatabaseVersion(Bytes.toIntUnsafe(ByteString(0, 1, 0, 0)))
+  val currentDBVersion: DatabaseVersion = DatabaseVersion(Bytes.toIntUnsafe(ByteString(0, 1, 1, 0)))
```
Comment (tdroxler) [inline flow/src/main/scala/org/alephium/flow/io/DatabaseVersion.scala:31]: we need to think how to merge and release that, as we will need to ask people to wipe out their db I guess?

Comment (Lbqds) [reply flow/src/main/scala/org/alephium/flow/io/DatabaseVersion.scala:31]: user can upgrade without any changes, but the downgrade needs to modify the database, we can provide a python script to do this.

**Review by polarker — APPROVED** [review]
LGTM. Recommend to rename `v1.1.0_to_v1.0.0.py` as `db_v1.1.0_to_v1.0.0.py`

**Review by tdroxler — APPROVED** [review]
Migration tool tested, it works like a charm :+1:

**Pull request merged**